test/fixtures/payload-helpers/features/traces_support.feature

```diff
@@ -10,4 +10,12 @@ Feature: Testing gzip support on traces endpoint
         When I run the script "features/scripts/send_gzip.sh" synchronously
         And I wait to receive a trace
         And the trace payload field "hello" equals "world"
-        And the trace payload field "array" is a non-empty array
\ No newline at end of file
+        And the trace payload field "array" is a non-empty array
+
+    Scenario: The trace endpoint can identify a valid request
+        When I send a "valid" trace request
+        Then I wait to receive a trace
+
+    Scenario: The trace endpoint can identify an invalid request
+        When I send a "invalid" trace request
+        Then I wait to receive a trace
```

Comment:
Presumably you'll need do work some magic here to stop the scenario failing the test run?

Reply:
Yes - this is more for testing/output examples right now